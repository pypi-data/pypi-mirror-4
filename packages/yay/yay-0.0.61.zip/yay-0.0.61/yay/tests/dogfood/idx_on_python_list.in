
lookup: foo

a:
 - name: foo
   list:
     - value: 1
     - value: 2
     - value: 3

b: ${a[@.name = lookup][0]}

c: ${b.list[0].value}

