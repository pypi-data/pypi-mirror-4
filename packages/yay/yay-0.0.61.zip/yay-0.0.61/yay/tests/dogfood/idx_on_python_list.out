
lookup: foo

a:
 - name: foo
   list:
     - value: 1
     - value: 2
     - value: 3

b:
   name: foo
   list:
     - value: 1
     - value: 2
     - value: 3

c: 1

